scripts/integrationtests-wizard.sh

```diff
@@ -1,5 +1,6 @@
 cd "$(pwd)/integrationtests"
-
+TRAVIS_BUILD_DIR="${TRAVIS_BUILD_DIR:-$GITHUB_WORKSPACE}"
```

Comment:
TODO we will remove this when we are sure we can ditch travis completely

Reply:
travis.yml is deleted, shouldn't this be removed now, then?

other than that: LGTM

Reply:
we still dont know where we will go to